src/ResourceManager/Automation/Commands.Automation.Test/ScenarioTests/AccountTests.ps1

```diff
@@ -0,0 +1,99 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+$newAccountName='account-powershell-test'
+$existingResourceGroup='PowerShellTest'
```

Comment:
Is there a reason you are using an existing resource group in the test instead of creating a new one and destroying it at the end of the test? This will make it much more difficult to rerecord.

Reply:
Creating a blank resource group doesn't make things much harder to record. The problem with builtin resource group creation is New-AzureRmResourceGroup -Name $newResourceGroup -Location $location requires more permission than our default test credential give. 

I think it is better to leave this as is.

Reply:
@thomasyip-msft When rerecording, it is expected that all you need to do is set up you environment and press "run test".  Having this extra setup step requires devs to go in, look up what needs to be precreated (which right now is not clear unless you read through the actual test code), then precreate the resource, making sure it's in the right subscription with the right variables. If that resource already exists, they either need to delete it or change the test code itself.  Our team occasionally needs to go through and rerecord large numbers of tests, and small setup requirements like this makes that process much harder.  Please give you test credential permissions to create a resource group and create it in the test.